pkg/apis/serving/v1alpha1/inference_graph.go

```diff
@@ -45,6 +46,10 @@ type InferenceGraphSpec struct {
 	// Map of InferenceGraph router nodes
 	// Each node defines the router which can be different routing types
 	Nodes map[string]InferenceRouter `json:"nodes"`
+	// +optional
+	Resources corev1.ResourceRequirements `json:"resources,omitempty" protobuf:"bytes,8,opt,name=resources"`
```

Comment:
Not sure if we need the protobuf def as we did not define for the other graph fields, also 8 here means the ordering of the fields on container spec but we do not include any of the other fields.

Reply:
Addressed.